### PR TITLE
capstone project 1 

### PR DESCRIPTION
**Pull request by Afsheen96**
Capstone Project: ICFP Contest

In this PR, we added the following:

 mobile home page
 mobile about page
 featured speakers using dynamic HTML
 mobile menu
 responsive webiste

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://afsheen96.github.io/Portfolio)
+- [Live Demo Link](https://afsheen96.github.io/Capstone-project-module-1)
```
Comment (TadesseDev) [inline README.md:59]: - [ ] You have done great with the project @Afsheen96, But for such an exercise the README of the project is expected to contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      </div>
+    </section>
+  </main>
+<!-- -----------------------------footer-------------------------------- -->
+  <footer>
+    <div class="footer footer-home">
+      <img src="./imgs/logoimg.jpg" class="footer-logo" alt="Microverse logo">
+      <p class="copy-right">2023 | International Conference on Functional Programming.</p>
+    </div>
+  </footer>
```
Comment (TadesseDev) [inline index.html:129]: - [ ] Dear @Afsheen96, Your footer is almost identical to the original design, The issue I can see is the spacing/alignment between elements (the logo and the text). As the requirement, a logo is aligned to the left and text aligned to the right. Please made this arrangement. 
### Your design 
image
### Original design 
image

File: index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+menu.addEventListener('click', openMenu);
+menu.addEventListener('click', closeMenu);
+const speakers = document.getElementById('speakers-container');
+
+const speakerData = [
+
+  {
+    img: './imgs/speaker1.png',
```
Comment (TadesseDev) [inline index.js]: - [ ] You have already done a great job organizing your feature speakers section, But as the requirement states, we need six individual articles (two columns of three articles each), Please consider adding two more. 

### Your design 
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        <i class="fa-brands fa-facebook-f social-icon"></i>
+        <i class="fa-brands fa-twitter social-icon"></i>
+        <ul class="topnav-list">
+          <li class="top-links">English</li>
+          <li class="top-links">MyPage</li>
+          <li class="top-links">Logout</li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <img src="./imgs/logoimg.jpg" class="logo" alt="ICFP Logo">
```
Comment (TadesseDev) [inline about.html]: - [ ] Dear @Afsheen96, for better UX and semantic reasons, The logo better be a link to the homepage. It is conventional for the user to click on a page logo to navigate to the homepage. Please consider making your logo a hyperlink.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Afsheen96 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitHub Flow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 4 above)
Comment (Afsheen96) [reply about.html]: have applied the changings @TadesseDev

**Review by DuaneDave — APPROVED** [review]
Hi @Afsheen96 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ You did a good job with the UI
✔️ You followed best practices in writing your code

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**